src/pages/resultsView/survival/SurvivalTab.tsx

```diff
@@ -54,6 +58,18 @@ export default class SurvivalTab extends React.Component<ISurvivalTabProps, {}>
         }
     });
 
+    private makeSurvivalInfoIcon(survivalType: String): JSX.Element {
+        return (
+            <DefaultTooltip
+                placement='top'
+                overlay={`${survivalType} since time of sequencing`}
```

Comment:
We should use the description of the OS_STATUS or DFS_STATUS attribute for the tooltip.

Reply:
![image](https://user-images.githubusercontent.com/840895/58204354-1f203600-7caa-11e9-98f7-9b2b80b68170.png)

https://cbioportal-frontend-pr-2345.herokuapp.com/study/summary?id=coadread_tcga_pub

Reply:
Sure, will use the description from the attribute.